### PR TITLE
Azure CLI beta release docs

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/install-azure-cli-beta.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+ms.custom: devx-track-azurecli
+---
+
+# Install Azure CLI beta version
+
+A new beta version of the Azure CLI has been released that offers increased security for token cache, access tokens and SSL certificates.  This beta vesion can be installed in Windows, macOS and Linux environments and will stay in sync with the most recent release.
+
+> [!NOTE]
+>
+>  BREAKING CHANGES are introduced in this release.  Carefully read all [release notes](/cli/azure/release-notes-azure-cli?tabs=azure-cli#tabs=azure-cli-beta) prior to installation.
```
Comment (jiasli) [inline docs-ref-conceptual/install-azure-cli-beta.md:20]:
```suggestion
>  BREAKING CHANGES are introduced in this release.  Carefully read all [release notes](/cli/azure/release-notes-azure-cli?tabs=azure-cli-beta) prior to installation.
```

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 2]
```diff
@@ -7020,19 +7020,28 @@ You can report issues with nightly preview builds in the following ways:
...
+> [!NOTE]
+>
+>  BREAKING CHANGES are introduced in this release.  Carefully read all release notes prior to installation.
+>
+> The beta version does not guarantee product level quality so it should not be used in your production environment.
+
+* Azure CLI beta internally replaces [ADAL](https://github.com/AzureAD/azure-activedirectory-library-for-python) with [Azure Identity](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity) and [MSAL](https://github.com/AzureAD/microsoft-authentication-library-for-python). Existing ADAL token cache (`~/.azure/accessToken.json`) will be migrated to MSAL encrypted token cache automatically when any command requiring a credential is executed.
+
+* There are several changes to `az login`.  (Run `az login --help` for more details.)
+  * [BREAKING CHANGE] `~/.azure/accessToken.json` will no longer be created after a successful login. To get an access token, use [`az account get-access-token`](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az_account_get_access_token) instead.
```
Comment (jiasli) [inline docs-ref-conceptual/release-notes-azure-cli.md:7034]:
```suggestion
  * [BREAKING CHANGE] `~/.azure/accessToken.json` will no longer be created after a successful login. To get an access token, use [`az account get-access-token`](https://docs.microsoft.com/cli/azure/account?view=azure-cli-latest#az_account_get_access_token) instead.
```

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 3]
```diff
@@ -7020,19 +7020,28 @@ You can report issues with nightly preview builds in the following ways:
...
+  * [BREAKING CHANGE] `--use-cert-sn-issuer` argument is not supported.
+  * After logging in with a managed identity, all `clientId`, `objectId` and `resourceId` will be shown.
+  * Fix #13188: `az login` with managed identity indicating system assigned when the identity is user assigned.
+
+* [BREAKING CHANGE] Skip SSL verification via environment `ADAL_PYTHON_SSL_NO_VERIFY` has been removed. See [work behind a proxy](https://docs.microsoft.com/en-us/cli/azure/use-cli-effectively#work-behind-a-proxy) for trusting a self-signed root certificate.
```
Comment (jiasli) [inline docs-ref-conceptual/release-notes-azure-cli.md:7039]:
```suggestion
* [BREAKING CHANGE] Skip SSL verification via environment `ADAL_PYTHON_SSL_NO_VERIFY` has been removed. See [work behind a proxy](https://docs.microsoft.com/cli/azure/use-cli-effectively#work-behind-a-proxy) for trusting a self-signed root certificate.
```

File: docs-ref-conceptual/install-azure-cli-beta.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+To avoid overwriting your installed Azure CLI, we recommend installing the beta version in a [virtual environment](https://docs.python.org/3/tutorial/venv.html).
+
+1. Create a virtual environment
+
+   Navigate to the folder where you want to create the virtual environment, then run:
+
+   ```console
+   python -m venv <env_name>
+   ```
+
```
Comment (jiasli) [inline docs-ref-conceptual/install-azure-cli-beta.md:61]:
```suggestion
The executable can be `python3`, depending on the Linux distribution or the installed Python you are using.

```

File: docs-ref-conceptual/install-azure-cli-beta.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+   ```console
+   . $env/bin/activate
+   ```
+   ---
+   These commands can also be used to reactivate your virtual environment.
+
+1. Install Azure CLI beta
+
+   ```console
+   pip install --pre --extra-index-url https://azurecliprod.blob.core.windows.net/beta/simple/ azure-cli
```
Comment (jiasli) [inline docs-ref-conceptual/install-azure-cli-beta.md:81]: We need @fengzhou-msft's help to upload the beta package to a SAW-managed storage account https://azcliprod.blob.core.windows.net/ instead.

```suggestion
   pip install --pre --extra-index-url https://azcliprod.blob.core.windows.net/beta/simple/ azure-cli
```

**Comment by dbradish-microsoft** [comment]
My branch got out of sync.  It was easier to close this PR and open a new one with all updates.

**Pull request closed without merging**